src/test/e2e/tests/specs.test.ts

```diff
@@ -151,10 +154,42 @@ describe('Unverified user', () => {
       expect(await singleJusticeProcedurePage.getPageTitle()).toEqual('Single Justice Procedure cases');
     });
   });
+
+  describe('Sign In Page', () => {
+    const returnUrl = 'https://www.google.com';
+    it('should open sign-in page with \'How do you want to sign in\' title', async () => {
+      await signInPage.open('sign-in');
+      expect(await signInPage.getPageTitle()).toEqual('How do you want to sign in?');
+    });
+
+    it('should see 3 radio buttons', async () => {
+      expect(await signInPage.radioButtons).toBe(3);
+    });
+
+    describe('sign in page routing', async () => {
+      it('should select \'Sign in with My HMCTS\' option and navigate to the login page HMCTS page', async () => {
+        await signInPage.selectOption('SignInRadio1');
+        expect(await signInPage.clickContinueForRadio1()).toHaveHref(returnUrl);
+      });
+
+      it('should select \'Sign in with Common Platform\' option and navigate to the login page Common Platform page', async () => {
+        await signInPage.open('sign-in');
```

Comment:
if you put `singInPage.open` into `beforeEach` you don't need to call it every time within `it` block

Reply:
moved open to beforeEach